pages/basePage.tsx

```diff
@@ -54,6 +55,13 @@ const BasePage: NextPage<BasePageProps> = ({ children }) => {
                 wallets={WALLETS}
                 theme={"light"}
                 connectModal={{ size: "compact" }}
+                accountAbstraction={{
+                  chain: arbitrum, // the chain where your smart wallet will be or is deployed
+                  factoryAddress: String(
+                    process.env.NEXT_PUBLIC_FACTORY_ADDRESS,
+                  ),
+                  gasless: false, // enable or disable gasless transactions
+                }}
```

Comment:
2. enabled `AA` flag by deploying our `aa wallet factory contract` to Arbitrum.

>  gasless: false, // enable or disable gasless transactions
Will enable this in the upcoming PR!